pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -433,13 +431,14 @@ func TestPostFilter(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			p := DefaultPreemption{
-				fh:        f,
-				podLister: informerFactory.Core().V1().Pods().Lister(),
-				pdbLister: getPDBLister(informerFactory),
-				args:      *getDefaultDefaultPreemptionArgs(),
+			pi, err := New(context.Background(), getDefaultDefaultPreemptionArgs(), f, feature.Features{})
+			if err != nil {
+				t.Fatal(err)
+			}
+			p, ok := pi.(*DefaultPreemption)
+			if !ok {
```

Comment:
Do we need a test case to verify that customized `EligiblePods` and `OrderedPods` (different from the default one) work as expected?

Reply:
Yes that's a good point, I'll write up some tests that exercise customizing these.